pykrige/ok.py

```diff
@@ -520,7 +520,7 @@ def _exec_loop_moving_window(self, a_all, bd_all, mask, bd_idx):
 
         return zvalues, sigmasq
 
-    def execute(self, style, xpoints, ypoints, mask=None, backend='vectorized', n_closest_points=None):
+    def execute(self, style, x, y, mask=None, backend='vectorized', n_closest_points=None):

```

Comment:
The problem is this would break backward compatibility for someone calling this function `execute(style, xpoints=x, ypoints=y)`, so we can't make this change unfortunately.